CLI.md

```diff
@@ -652,6 +654,25 @@ Publish module
 
 
 
+## `linera list-events-from-index`
+
+Reading the events from a starting index
```

Comment:
This description isn't helpful as I have no idea what will happen when I run this command.

Should it say "prints events for <chain_id>, using "some" format. Optionally caller can specify a stream/starting index" ?


Also, what will happen when there are thousands of such events? Why do we need this command in the CLI?

Reply:
Ok, it has been renamed.
This is needed for the node service.